dds_web/commands.py

```diff
@@ -345,6 +345,397 @@ def lost_files_s3_db(action_type: str):
+                    except (
+                        sqlalchemy.exc.OperationalError,
+                        sqlalchemy.exc.SQLAlchemyError,
+                    ) as err:
+                        flask.current_app.logger.exception(err)
+                        db.session.rollback()
+                        errors[unit.name][project.public_id] = str(err)
+                    continue
+                else:
+                    flask.current_app.logger.debug(
+                        "Nothing to do for Project: %s", project.public_id
+                    )
+    except (sqlalchemy.exc.OperationalError, sqlalchemy.exc.SQLAlchemyError) as err:
+        flask.current_app.logger.exception(err)
+        db.session.rollback()
+        raise
+
+    for unit, projects in errors.items():
+        if projects:
+            flask.current_app.logger.error(
+                f"Following projects of Unit '{unit}' encountered issues during archival process:"
+            )
+            for proj in errors[unit].keys():
+                flask.current_app.logger.error(f"Error for project '{proj}': {errors[unit][proj]} ")
+
+
+@click.command("delete-invites")
+@flask.cli.with_appcontext
+def delete_invites():
+    """Delete invites older than a week"""
+
+    flask.current_app.logger.debug("Task: Checking for invites to delete.")
+
+    # Imports
+    # Installed
+    from datetime import datetime, timedelta
+    from sqlalchemy.exc import OperationalError, SQLAlchemyError
+
+    # Own
+    from dds_web import db
+    from dds_web.database import models
+    from dds_web.errors import DatabaseError
+    from dds_web.utils import current_time
+
+    expiration: datetime.datetime = current_time()
+    errors: Dict = {}
+
+    try:
+        invites: list = db.session.query(models.Invite).all()
+        for invite in invites:
+            invalid_invite = invite.created_at == "0000-00-00 00:00:00"
+            if invalid_invite or (invite.created_at + timedelta(weeks=1)) < expiration:
+                try:
+                    db.session.delete(invite)
+                    db.session.commit()
+                    if invalid_invite:
+                        flask.current_app.logger.warning(
+                            "Invite with created_at = 0000-00-00 00:00:00 deleted."
+                        )
+                    else:
+                        flask.current_app.logger.debug("Invite deleted.")
+                except (OperationalError, SQLAlchemyError) as err:
+                    errors[invite] = str(err)
+                    flask.current_app.logger.exception(err)
+                    db.session.rollback()
+                    continue
+    except (OperationalError, SQLAlchemyError) as err:
+        flask.current_app.logger.exception(err)
+        raise
+
+    for invite, error in errors.items():
+        flask.current_app.logger.error(f"{invite} not deleted: {error}")
+
+
+@click.command("quartely-usage")
+@flask.cli.with_appcontext
+def quarterly_usage():
```

Comment:
Add docstring with information on how often it runs and when. Get from apscheduler.